### PR TITLE
hardcode expired banner text for G11

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/26eeD0Pd/1949-g11-banner-content-change
Hardcoding this banner for G11 for now, ideally should be derived from the framework object
Screenshot 2020-10-08 at 16 53 19

### CODE REVIEW

File: app/templates/service.html [hunk 1]
```diff
@@ -32,13 +32,22 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     {% if service_unavailability_information.type == 'framework_update' %}
+    {# TODO: Hardcoding this banner for G11 for now, should be derived from the framework object  #}
+       {% if service.frameworkName == 'G-Cloud 11' %}
+           {{ dmBanner({
+        "title": "This G-Cloud 11 service has been suspended and is no longer searchable on the Digital Marketplace.",
+        "text": "The G-Cloud 11 framework expires on 18 December 2020 to allow in progress procurements to be concluded. Existing contracts with {} are still valid.".format(
```
Comment (bjgill) [inline app/templates/service.html:39]:
```suggestion
        "text": "The G-Cloud 11 framework expires on 18 December 2020 to allow procurements in progress to be concluded. Existing contracts with {} are still valid.".format(
```

**Review by bjgill — COMMENTED** [review]
Looks good. Just a wording suggestion

File: app/templates/service.html (hunk 1 above)
Comment (bjgill) [reply app/templates/service.html:39]: Or should we be more assertive? "Procurements in progress to be concluded before the G-Cloud 11 framework expires on 18 December 2020."?

Comment (gidsg) [reply app/templates/service.html:39]: I feel like the wording is really one for @NoraGDS who's not in today. Can we use this for now and then can revise when @NoraGDS is back?

File: app/templates/service.html [hunk 2]
```diff
@@ -32,13 +32,22 @@
...
+      }) }}
+   {% else %}
       {{ dmBanner({
         "title": "This {} service is no longer available to buy.".format(service.frameworkName),
         "text": "The {} framework expired on {}. Any existing contracts with {} are still valid.".format(
             service.frameworkName,
             framework_expires_at_date,
             service.supplierName)
       }) }}
+   {% endif %}
```
Comment (bjgill) [inline app/templates/service.html:50]: Is this indentation correct? It doesn't seem to line up with the `if` to me...

File: app/templates/service.html (hunk 1 above)
Comment (phantas) [reply app/templates/service.html:39]: who is the source for this content? I agree with the change in any case. I'd still like Nora's eyes on this but I am happy to go ahead for now and change later if needed.

Comment (bjgill) [reply app/templates/service.html:39]: Oh, I see this wording came from CCS. Never mind, then. Happy for this content to go in as-is and let a content professional to take a look tomorrow.

**Review by phantas — APPROVED** [review]

File: app/templates/service.html (hunk 2 above)
Comment (phantas) [reply app/templates/service.html:50]: it is correct

File: app/templates/service.html [hunk 3]
```diff
@@ -32,13 +32,22 @@
...
     {% else %}
```
Comment (phantas) [inline app/templates/service.html:51]: this however isn't.

File: app/templates/service.html (hunk 2 above)
Comment (gidsg) [reply app/templates/service.html]: OK I don't know, I got intellij to indent it and committed that in 49bc491

File: app/templates/service.html (hunk 3 above)
Comment (gidsg) [reply app/templates/service.html:51]: should be fixed in 49bc491

File: app/templates/service.html (hunk 1 above)
Comment (gidsg) [reply app/templates/service.html:39]: yes, source is CCS verbatim for now.

**Review by bjgill — APPROVED** [review]

**Pull request merged**